electronics_model/KiCadSchematicBlock.py

```diff
@@ -0,0 +1,52 @@
+from kinparse import parse_netlist  # type: ignore
+from edg_core import Block
+from electronics_abstract_parts import Resistor, Capacitor
+from electronics_model import Ohm
+
+
+class KiCadSchematicBlock(Block):
+
+    def import_kicad(self, filepath: str):
+
+        netlist = parse_netlist(filepath)
+
+        for part in netlist.parts:
+            setattr(self, part.ref, self.make_block_from_mapping(part))
+
+        for net_num, net in enumerate(netlist.nets):
+            portlist = []
+            for pin in net.pins:
+                component = getattr(self, pin.ref)
+                if isinstance(component, Resistor):
+                    if pin.num == "1":
+                        portlist.append(component.a)
+                    else:
+                        portlist.append(component.b)
+                if isinstance(component, Capacitor):
```

Comment:
consider making this an `elif` - it shouldn't be possible for both cases to be true